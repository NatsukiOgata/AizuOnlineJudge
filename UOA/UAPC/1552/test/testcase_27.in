1000 12 14 13 3 6 5 10 19 4 11 1 7 19 18 1 17 14 18 9 8 17 10 15 3 1 8 6 10 6 17 14 7 2 14 10 3 0 12 10 11 8 9 7 1 10 2 13 14 3 4 4 11 8 17 10 10 4 1 18 1 2 3 11 12 8 7 12 5 16 12 12 0 6 15 4 12 19 8 10 4 0 4 9 12 16 5 6 14 13 14 17 1 2 9 12 13 4 2 18 4 0 5 10 12 14 2 10 6 14 7 16 3 19 10 8 14 7 0 8 5 13 10 16 9 3 2 11 5 4 18 2 18 10 18 16 5 9 1 7 18 2 4 0 17 13 11 9 5 11 14 15 3 12 13 11 14 10 19 13 12 0 2 12 15 2 12 3 15 16 12 1 5 5 9 15 11 19 14 2 19 1 6 12 7 0 5 4 15 11 2 17 4 6 17 15 8 10 19 14 14 13 19 18 5 8 12 4 12 18 6 7 8 7 12 1 15 5 12 8 12 11 10 3 0 2 7 5 9 9 10 2 9 19 3 0 6 14 17 2 0 6 12 18 6 15 2 12 4 15 18 8 13 7 16 8 11 19 11 9 1 18 10 14 13 4 12 1 1 16 12 6 10 11 13 0 2 1 15 9 3 7 13 9 10 18 8 2 2 14 6 2 15 17 2 12 14 4 16 13 8 3 5 16 3 14 11 11 0 2 6 12 7 5 7 9 11 11 7 8 16 11 19 12 5 10 9 17 14 17 14 5 5 3 17 18 6 0 9 15 14 13 0 2 14 16 3 16 19 12 11 0 8 9 19 8 0 11 1 19 13 14 13 18 8 5 1 0 9 14 14 0 12 16 17 15 8 14 14 1 0 6 12 7 12 15 15 5 19 11 12 12 13 3 13 8 19 14 19 19 16 11 12 9 7 8 18 12 3 0 8 8 1 4 6 7 12 3 16 9 1 1 18 0 12 3 13 19 18 8 8 5 8 19 0 9 17 2 2 16 5 15 6 0 16 6 9 1 13 16 4 18 12 10 4 17 8 0 18 7 5 1 0 0 17 17 9 1 2 18 11 15 1 1 15 2 16 18 8 2 14 8 10 17 10 2 7 16 10 1 10 0 17 4 14 19 17 10 6 13 10 18 4 13 3 8 15 7 5 7 7 0 2 14 13 13 2 2 9 9 2 19 16 17 3 13 16 6 19 1 15 2 5 3 2 13 4 15 15 19 15 19 11 0 6 7 19 8 5 15 18 0 19 14 19 12 0 13 4 19 18 17 14 5 18 16 12 3 3 11 11 8 16 13 5 0 12 6 17 4 10 3 19 12 15 17 9 10 8 12 6 16 7 15 0 7 9 18 14 14 12 18 14 12 18 15 0 17 3 14 8 7 3 8 19 16 14 12 4 1 18 18 0 18 14 1 3 3 5 18 8 10 8 9 5 6 5 5 12 0 18 10 10 8 14 15 14 19 8 1 15 14 0 9 16 18 6 9 8 14 14 9 3 11 0 7 6 1 1 15 9 2 18 6 8 13 3 15 16 9 14 14 19 15 7 10 7 10 17 1 17 10 2 18 3 0 9 11 1 12 11 11 1 7 2 9 7 6 17 12 2 13 19 7 15 13 9 8 4 8 5 9 13 17 2 16 15 12 3 1 2 6 5 16 6 3 19 14 16 9 1 11 8 6 1 6 5 7 14 2 7 15 6 17 17 3 12 12 3 11 9 4 14 17 17 2 7 1 9 13 14 1 2 12 16 12 1 1 11 4 14 6 16 18 10 14 15 12 12 8 17 3 3 14 4 10 18 7 14 17 13 1 10 15 19 9 1 6 9 18 1 18 8 1 15 2 2 4 10 13 17 11 12 9 10 13 5 5 12 12 3 3 10 16 6 3 0 10 10 12 10 13 17 6 15 4 6 9 10 14 0 14 16 0 11 0 2 17 1 18 13 15 13 15 14 17 7 7 12 10 13 7 8 7 0 14 11 17 14 8 15 1 12 18 17 6 5 5 17 6 8 6 5 17 8 10 8 6 8 17 5 13 4 18 9 5 8 14 17 16 2 19 9 19 3 18 3 17 5 16 3 17 13 2 18 8 3 16 1 0 14 1 14 1 18 2 11 12 7 1 9 6 17 4 4 11 8 5 2 16 1 18 18 2 18 5 15 6 8 14 10 15 7 1 6 17 16 18 11 6 9 9 4 17 1 6 19 11 18 0 16 12 17 14 19 6 14 16 13 15 16 5 5 8 0 18 16 2 16 6 12
