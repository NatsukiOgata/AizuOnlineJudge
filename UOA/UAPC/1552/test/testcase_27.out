18 19 19 304 303
